Realm/ObjectServerTests/RLMObjectServerTests.m

```diff
@@ -20,6 +20,25 @@
 
 #import "RLMSyncUser+ObjectServerTests.h"
 
+static NSString *reduceMethodName(const char *function) {
```

Comment:
doesn't seem like there's any reason to have this take `const char *` rather than `NSString *`, and just do the `@()` wrapping at the call site / macro if needed.